war/src/main/js/add-item.js

```diff
@@ -82,6 +82,27 @@ $.when(getItems()).done(function(data) {
       $('.input-help', context).removeClass('input-message-disabled');
     }
 
+    var doSticky = function() {
+      var s = $('form .footer .btn-decorator');
```

Comment:
Maybe it's a common pattern in JS, but shouldn't we use a longer/more explicit variable name than `s`?